src/Avalonia.X11/X11Window.cs

```diff
@@ -342,10 +342,17 @@ void OnEventSync(XEvent ev)
             {
                 if (ActivateTransientChildIfNeeded())
                     return;
-                if (ev.ButtonEvent.button < 4)
-                    MouseEvent(ev.ButtonEvent.button == 1 ? RawPointerEventType.LeftButtonDown
-                        : ev.ButtonEvent.button == 2 ? RawPointerEventType.MiddleButtonDown
-                        : RawPointerEventType.RightButtonDown, ref ev, ev.ButtonEvent.state);
+                if (ev.ButtonEvent.button < 4 || ev.ButtonEvent.button == 8 || ev.ButtonEvent.button == 9)
```

Comment:
On systems with modern X server mouse input is handled by [XI2Manager](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.X11/XI2Manager.cs) class.